packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -387,7 +387,7 @@ This can be used only together with build type ${input('release')} for Android a
     if (!filename) {
       req.on('response', res => {
         const contentDisposition = res.header['content-disposition'];
-        filename = contentDisposition ? contentDisposition.split('=')[1] : 'output.bin';
+        filename = contentDisposition ? contentDisposition.split('=')[1].replace(/([/,?,<,>,*,|,\"])/g, '_') : 'output.bin';
```

Comment:
Regular expression character sets don't use any separators:

```suggestion
        filename = contentDisposition ? contentDisposition.split('=')[1].replace(/([/?<>*|\"])/g, '_') : 'output.bin';
```

Reply:
ok. I defer to you on regex.